packages/webapp/src/containers/Task/styles.module.scss

```diff
@@ -0,0 +1,21 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+.buttonWrapper {
+  position: fixed;
+  bottom: 16px;
+  right: 16px;
+  z-index: 100;
```

Comment:
I wonder if we should set a z-index on the floating button component itself since it's supposed to always float over everything else?

Reply:
That makes sense to me. Do you think the positioning CSS should even be controlled by a prop on the component if we're using it so often in this corner? 🤔

Reply:
Oh yes, that's a really good idea!